impl/src/main/java/org/jboss/weld/Container.java

```diff
@@ -208,4 +215,20 @@ public void setState(ContainerState state) {
         this.state = state;
     }
 
+    /**
+     * Returns {@link Environment} that this container was booted in, or null if not set or invoked too
```

Comment:
We should be clear that this method returns the env the **first** container was started in.

Reply:
Should be, not needn't be true if you start it and not set environment (using one of the older methods, not the new one I added).
I will add this to the doc.